packages/features/calendars/DatePicker.tsx

```diff
@@ -240,9 +230,7 @@ const Days = ({
         </div>
       ))}
 
-      {!props.isLoading && includedDates && includedDates?.length === 0 && (
-        <NoAvailabilityOverlay month={month} nextMonthButton={nextMonthButton} />
-      )}
```

Comment:
why are we deleting this? We wanna keep this in case people don't have availabilities in a month

Reply:
Because @PeerRich sir, this is related to the limit future booking setting:

<img width="634" alt="image" src="https://github.com/calcom/cal.com/assets/56840292/59719ba8-1d02-442c-9aee-f1120fb4f128">

For example if it allows booking 10 days in the future then it is sure that there will be no availability anytime in the future after 10 days.

So instead of showing this to the user which have a button to view next month which is misleading because of the same reason mentioned above (i.e. there will be no availability in the future too).

<img width="482" alt="image" src="https://github.com/calcom/cal.com/assets/56840292/f9097533-c897-4e9e-ac18-d668e4e43052">

we should show this:

![image](https://github.com/calcom/cal.com/assets/56840292/80a4413d-c003-486b-9666-fee6e2edd077)

Because sir, view next month says that "go to next month there may be availabilities in the next month" but actually it is not true.

Reply:
ok but would this change not remove this functionality all together, lets say if you have no free times in October, but you have them in November? would we still show the button? because we want to show the button then